R/stars.R

```diff
@@ -132,7 +132,7 @@ leaflet_stars = function(x,
     if (is.fact) {
       ### delete at some stage!!! ###
       pal = leaflet::colorFactor(palette = col.regions,
-                                 domain = seq(0, 255, 1),
+                                 domain = seq(1, length.out = length(levels(x[[1]]))),
```

Comment:
Is it possible that the data values are rescaled to be 0-based? The legend shows `land["cover"]-1` - does the `- 1` indicate this?